JS/Lesson22/goksel/style.css

```diff
@@ -0,0 +1,105 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    font-family: sans-serif;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background-color: #000
+}
+
+.content-header {
+    text-align: center;
+    font-size: 24px;
+    padding: 12px;
+    margin: 5px;
+    font-weight: 600;
+}
+
+.accordion {
+    max-width: 600px;
+    background-color: #fff;
+    border-radius: 5px;
+    padding: 10px;
+}
+
+.accordion-container{ 
+    border: 1px solid #000;
+    border-radius: 5px;
+    margin: 5px;
+}
+
+
+
+.click-btn {
+display: flex;
+align-items: center;
+justify-content: space-between;
```

Comment:
There's only one child of the 'click-btn' class which mean that the 'justify-content: space-between;' property would not really work. Maybe you want to use 'stretch' or 'center'?

Reply:
ı used "justfiy-content:space-between;" because ı wanted to put "+" sign to the right side. And ı added "+" sign .click-btn::after so ı was working like that. Should ı still change it?

Reply:
No, that's fine in this case!